translate/filters/pofilter.py

```diff
@@ -31,19 +31,27 @@
 
 import os
 import six
+import sys
+from importlib import import_module
 
 from translate.filters import autocorrect, checks
 from translate.misc import optrecurse
 from translate.storage import factory
 from translate.storage.poheader import poheader
 
 
-def build_checkerconfig(options):
+def build_checkerconfig(self, options):
```

Comment:
Actually it doesn't seem like this would work in the case of the test since `BaseTestFilter` does not have `error` method.